src/libraries/Directory.Build.props

```diff
@@ -169,8 +170,8 @@
     <BaseIntermediateOutputPath Condition="'$(IsReferenceAssembly)' == 'true'">$([MSBuild]::NormalizeDirectory('$(ArtifactsObjDir)', '$(MSBuildProjectName)', 'ref'))</BaseIntermediateOutputPath>
     <BaseOutputPath Condition="'$(IsReferenceAssembly)' == 'true'">$([MSBuild]::NormalizeDirectory('$(AssemblyBinDirOutputPath)', 'ref'))</BaseOutputPath>    
   </PropertyGroup>
-  
-  <Import Project="$(MSBuildThisDirectory)targetframework.props" Condition="'$(DesignTimeBuild)' != 'true'" />
+
+  <Import Project="$(MSBuildThisDirectory)targetframeworksuffix.props" Condition="'$(DesignTimeBuild)' != 'true'" />
```

Comment:
Why do we need to import that file in Directory.Build.props?

Reply:
Because this file sets TargetsWindows, TargetsUnix properties which are being used in *.csprojs

Reply:
Can we change the csproj files to use TargetFrameworkSuffix instead?

Reply:
Disregard my comment.